src/components/Calculator.scss

```diff
@@ -0,0 +1,71 @@
+.space {
+  margin-top: 50px;
+}
+
+.container {
+  margin: auto;
+  width: 30%;
+  height: 70%;
+  padding: 10px;
+  border: 1px solid #f2f2f2;
+  -webkit-box-shadow: 1px 1px 4px 0 rgba(176, 176, 176, 1);
+  -moz-box-shadow: 1px 1px 4px 0 rgba(176, 176, 176, 1);
+  box-shadow: 1px 1px 4px 0 rgba(176, 176, 176, 1);
+  background-color: #f8f8f8;
+}
+
+.result1 {
+  height: 40px;
+  background-color: gray;
+
+  /*   padding: 10px; */
+  color: white;
+  text-align: right;
+  font-size: 20px;
+}
+
+table {
+  width: 100%;
+}
+
+td {
+  width: 25%;
+}
+
+.result {
+  font-size: 24px;
+  padding: 16px 5px;
+  font-weight: 700;
+  text-align: right;
+  width: 96%;
+  border: none;
+  background-color: #f8f8f8;
+}
+
+.global {
+  // padding: 15px;
+  font-weight: bold;
+  font-size: 16px;
+  border: none;
+  border-radius: 3px;
+  background-color: #f2f2f2;
+  width: 100%;
+  height: 70px;
+}
+
+/* the global class padding reset for first-row class button */
+.first-row {
+  padding: 7px;
+}
+
+.grey {
+  background-color: #f2f2f2;
+}
+
+#plus,
+#slash,
+#ex,
+#minus,
+#equal {
+  background-color: bisque;
+}
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A), Please change the background color of the operators buttons here to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/218886466-d6063c5f-beac-42dc-bfd6-26f1d4aa1b1f.png)  | ![image](https://user-images.githubusercontent.com/68796232/218886386-777141cc-9750-481d-a716-2c8dd80bda77.png) |

Hint: You can use this color `#F5913E`

Reply:
@Omar-Muhamad Thanks for your suggestion and especially hint 💯